webrtc/idlharness.https.window.js

```diff
@@ -5,7 +5,7 @@
 'use strict';
 
 // The following helper functions are called from RTCPeerConnection-helper.js:
-//   generateAnswer()
+//   generateDataChannelAnswer()
```

Comment:
`generateAnswer()` is not renamed to `generateDataChannelAnswer()`.

Reply:
Fixed in above commit